src/main/java/org/opensearch/agent/tools/PPLTool.java

```diff
@@ -356,6 +358,32 @@ private GetMappingsRequest buildGetMappingRequest(String indexName) {
         return getMappingsRequest;
     }
 
+    private static void validatePPLToolParameters(Map<String, Object> map) {
```

Comment:
Looks like this if...else can be merged to `StringUtils.isEmpty(map.get("model_id"))`?